es-app/src/main.cpp

```diff
@@ -332,8 +356,12 @@ int main(int argc, char* argv[])
 		lastTime = curTime;
 
 		// cap deltaTime at 1000
-		if(deltaTime > 1000 || deltaTime < 0)
+		if(deltaTime > timeout || deltaTime < 0)
 			deltaTime = 1000;
+			
+		t %= 4;
+		printf("\r%c", tes[t++]);
```

Comment:
Is this really necessary or is it just for debugging?
I found it a little weird when I saw that char "rolling" on the terminal.